Makefile

```diff
@@ -0,0 +1,17 @@
+BLDDIR = build
+
+# Necessary because zip copies leading directories if run from above targets
+ABS_BLDDIR := $(shell readlink -f $(BLDDIR))
+
+all: xpi
+
+xpi: $(BLDDIR)/tbkeys.xpi
+
+$(BLDDIR)/tbkeys.xpi:
+	@mkdir -p $(dir $@)
+	cd addon; zip -FSr $(ABS_BLDDIR)/tbkeys.xpi * -x \*.swp -x '#*#' -x \*~
```

Comment:
why not be consistent and always use `""` (or `''`) for shell quoting for glob chars?

Reply:
I copied this from another of my projects, so I don't remember exactly. I think I was using `\*.swp` because that matches the syntax recommended in the `zip` man page. For `#*#` every character has to be escaped, so that was excessive. Also, vim at least doesn't handle `\#\*\#` properly in a Makefile (it colors everything after the first `#` as a comment). I switched everything to single quotes.